docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Using Teleport CA with Github.
+description:  How to use Teleport SSH short lived certificates with the GitHubs Certificate Authority.
+---
+
+Teleport supports exporting user SSH certificates with configurable key extensions, this allows the Teleport CA to be used in conjunction with GitHubs support for SSH Certificate Authorities to allow users to access their organizations repositories with short lived, signed SSH keys.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
+- Telport with [GitHub SSO configured](../admin/github-sso.mdx).
+- Access to GitHub enterprise and permisions to modify GitHub SSH Certificate authorities.
+
+## Step 1/3. Import the Teleport CA into GitHub
+
+In order to export the Teleport CA, execute the following command:
+
+```code
+$ tctl auth export --type=user | sed 's/^cert-authority //g'
+```
+
+Next follow the instructions in the guide below to import your Teleport CA into GitHUB
+
+[Managing your organization's SSH certificate authorities](https://docs.github.com/en/organizations/managing-git-access-to-your-organizations-repositories/managing-your-organizations-ssh-certificate-authorities)
+
+The contents of the exported `teleport.ca` file should by pasted into the "Key" field after clicking "New CA".
+
+## Step 2/3. Configure the GitHub key extension
+
+Create or update a role to include the `cert_extensions` option, in order to interact with the support GitHub has for  SSH CAs, the name value here must be `login@github.com`.
```

Comment:
```suggestion
Create or update a role to include the `cert_extensions` option. The value of `name` must be `login@github.com`.
```
Suggested tightening this sentence up by removing a detail the guide has already established.